pkg/apis/v1alpha2/labels.go

```diff
@@ -67,7 +67,7 @@ var (
 		LabelSKUHyperVGeneration,
 	)
 
-	AllowUndefinedLabels = func(options scheduling.CompatabilityOptions) scheduling.CompatabilityOptions {
+	AllowUndefinedLabels = func(options scheduling.CompatibilityOptions) scheduling.CompatibilityOptions {
```

Comment:
I think this will cause an issue. I've found that this gets automatically auto-corrected if you commit from a code space. 

However, unless the misspelling has been updated in the core dep, than this reference will be wrong.

Reply:
https://github.com/Azure/karpenter-provider-azure/pull/112 the description of this PR claims the other way so we changed it.

Reply:
https://github.com/Azure/karpenter-provider-azure/blob/df78996ec2d6e2335b88dc941c5bf7aceb0733f8/typos we can add it to this list, but according to that PR we shouldn't be see this as a typo. 

Also Merriam Webster https://www.merriam-webster.com/thesaurus/compatibility

Reply:
Ok, I see what I was missing. The typo did get updated within core:
- https://github.com/kubernetes-sigs/karpenter/commit/80b052c4962512cb0a13457e8ed758f163801912

Reply:
Ah i see what you mean now. It was autofixing before.